Fixes failure when rebuilding build.xml with new ant:

 * Applying buildxml.patch ...
 * Applying buildxml-ant.patch ...
>>> Source unpacked in /var/tmp/portage/net-im/openfire-3.7.1/work
>>> Compiling source in /var/tmp/portage/net-im/openfire-3.7.1/work/openfire_src ...
Rewriting attributes
Rewriting ./build/build.xml
Rewriting ./src/test/throttletest/build/build.xml
Rewriting ./src/plugins/kraken/build/build.xml
 * Using following ANT_TASKS: ant-contrib
Buildfile: openfire-3.7.1/work/openfire_src/build/build.xml

init:

BUILD FAILED
openfire-3.7.1/work/openfire_src/build/build.xml:221: Must use JDK 1.5.x or higher to build Openfire

Gentoo-bug: https://bugs.gentoo.org/415619
Author: Matt McAdoo
--- build/build.xml	2012-05-15 21:15:53.849153950 -0500
+++ build/build.xml	2012-05-15 21:16:23.628271564 -0500
@@ -234,6 +234,7 @@
                 <or>
                     <contains string="${ant.java.version}" substring="1.5"/>
                     <contains string="${ant.java.version}" substring="1.6"/>
+                    <contains string="${ant.java.version}" substring="1.7"/>
                 </or>
             </not>
         </condition>
